### DISCUSSION TITLE
I deleted  package-lock.json thousand years ago，but I still got dependabot alerts every week.

### DISCUSSION THREAD

**Discussion by fyzhu** [community/community#41731] [Code Security] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/5175751/207605919-760093bc-e568-45a2-a826-5b608979f5f8.png)

https://github.com/fyzhu/935vue-todo

**Comment by blaufish** [comment, marked as answer]
Several projects experiencing this.

there is a strange workaround; 

- create dummy file in one commit. Some claims the dummy files need to be empty. 
- delete dummy file in next commit

https://github.com/dependabot/dependabot-core/issues/2041
